Makefile

```diff
@@ -234,6 +234,7 @@ release-cli: clean-debug build-ui
 	make BIN_NAME=argocd-linux-amd64 GOOS=linux argocd-all
 	make BIN_NAME=argocd-linux-arm64 GOOS=linux GOARCH=arm64 argocd-all
 	make BIN_NAME=argocd-windows-amd64.exe GOOS=windows argocd-all
+	make BIN_NAME=argocd-aix-ppc64 GOOS=aix GOARCH=ppc64 argocd-all
```

Comment:
Can you consider adding these to your PR?

```
make BIN_NAME=argocd-linux-ppc64le GOOS=linux GOARCH=ppc64le argocd-all
make BIN_NAME=argocd-linux-s390x GOOS=linux GOARCH=s390x argocd-all
```

Reply:
@krishvoor fixed